Rakefile

```diff
@@ -18,6 +18,35 @@ REQUIRED_ENVS = %w{AZURE_CLIENT_ID AZURE_CLIENT_SECRET AZURE_TENANT_ID AZURE_SUB
 
 task default: :test
 desc 'Testing tasks'
+
+task test: %w{test:unit setup_env test:integration}
```

Comment:
It appears that a test task has recently been added to the rakefile. Would you be able to remove these changes?

Reply:
removed